src/libs/MyNotifications.ts

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+
+import { MyNotify } from '@/types/api/NotificationType';
+
+import api from './textAxios';
+
+export const getMyNotifications = async (
+  cursorId?: number,
+  size?: number,
+): Promise<MyNotify> => {
```

Comment:
cursorId와 size는 쿼리인걸로 알고 있습니다!
저희 오늘 데일리 미팅때 상의했던 대로 객체로 묶어주시면 될것같아요 🙂
```suggestion
export const getMyNotifications = async (
  query?: { cursorId?: number; size?: number }
): Promise<MyNotify> => {
```

Reply:
넵 감사합니다~